blog/real-time-insights-on-large-volume-of-email-data-for-saas-crm.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Real-time insights on large volume of email data for SaaS CRM
```

Comment:
SaaS Insight for Building a Real-time CRM Application?

Reply:
> 

I think 'SaaS Insight for Building a Real-time CRM Application' is better but Xiaoyu Ma still use this title for card... Can I change it ?